cli/flags.rs

```diff
@@ -84,7 +84,9 @@ pub enum DenoSubcommand {
     root: Option<PathBuf>,
     force: bool,
   },
-  Lsp,
+  Lsp {
+    parent_pid: Option<u32>,
```

Comment:
Used `u32` to align with what Rust uses for pids: https://doc.rust-lang.org/std/process/struct.Child.html#method.id